src/EncodingOption.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Filter;
+
+use Laminas\Filter\Exception\InvalidArgumentException;
+
+use function array_map;
+use function in_array;
+use function mb_internal_encoding;
+use function mb_list_encodings;
+use function sprintf;
+use function strtolower;
+
+/** @internal */
+final class EncodingOption
```

Comment:
This class is marked as final and internal, should the constructor then be private or is this irrelevant here?

Reply:
I don't see it as relevant - if someone wants to `new EncodingOption()` on an `@internal` class - leave them to it! I can't see a reason to prevent instantiation when it's obviously a wrapper for static methods.